certified-connectors/AzureDigitalTwins/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1002 @@
+                  "type": "string",
+                  "description": "Link to get next set of items.",
+                  "title": "Continuation Token"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Query API",
+        "description": "Uses raw query to return results.",
+        "operationId": "QueryTwins",
+        "parameters": [
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "default": "2020-10-31",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "query": {
+                  "type": "string",
+                  "description": "Query the Twins."
+                },
+                "continuationToken": {
+                  "type": "string",
+                  "description": "Link to get next set of items."
+                }
+              }
+            }
+          }
+        ]
+      }
+    }
+  },
+  "definitions": {
+    "IncomingRelationship": {
+      "type": "object",
+      "properties": {
+        "$sourceId": {
+          "type": "string",
+          "description": "Source Id."
+        },
+        "$relationshipId": {
+          "type": "string",
+          "description": "Relationship Id."
+        },
+        "$relationshipName": {
+          "type": "string",
+          "description": "Relationship Name."
+        },
+        "$relationshipLink": {
+          "type": "string",
+          "description": "Relationship Link."
+        }
+      }
+    },
+    "TwinRelationship": {
+      "type": "object",
+      "properties": {
+        "$sourceId": {
+          "type": "string",
+          "description": "Source Id."
+        },
+        "$relationshipId": {
+          "type": "string",
+          "description": "Relationship Id."
+        },
+        "$targetId": {
+          "type": "string",
+          "description": "Target Id."
+        },
+        "$relationshipName": {
+          "type": "string",
+          "description": "Relationship Name."
+        },
+        "$etag": {
+          "type": "string",
+          "description": "E Tag Link."
+        },
+        "additionalProperties": {
+          "type": "string",
+          "description": "Property Values."
+        }
+      }
+    }
+  },
+  "parameters": {
+    "integerUpdate": {
+      "name": "body",
+      "in": "body",
+      "required": false,
+      "schema": {
+        "type": "array",
+        "items": {
+          "type": "object",
+          "properties": {
+            "op": {
+              "type": "string",
+              "description": "Request Operation."
+            },
+            "path": {
+              "type": "string",
+              "description": "Path that need to be updated."
```

Comment:
@sankumrav please make sure that descriptions are grammatically correct and are free of typographical errors.